<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmCadastroUsuario</class>
 <widget class="QWidget" name="frmCadastroUsuario">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>420</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>401</width>
    <height>420</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>401</width>
    <height>420</height>
   </size>
  </property>
  <property name="font">
   <font>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Cadastro de Usuário</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#frmCadastroUsuario
{
	
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(85, 0, 127, 255), stop:1 rgba(137, 0, 206, 255));
   font: &quot;URW Gothic L&quot;;
}

#gpbUsuario
{
	background-color: rgba(0, 0, 0, 50);
}

#btnCancelar
{
	background-color: rgb(106, 0, 0);
}

#btnLimpar
{	
	background-color: rgb(89, 89, 89);
}

#btnConcluir
{
	background-color: rgb(0, 85, 0);
}</string>
  </property>
  <widget class="QLabel" name="lblCadastroUsuario">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>321</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>321</width>
     <height>23</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>URW Gothic L</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>CADASTRO DO USUÁRIO</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancelar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>101</width>
     <height>41</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>URW Gothic L</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnConcluir">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>370</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>101</width>
     <height>41</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>URW Gothic L</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Concluir</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLimpar">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>370</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>101</width>
     <height>41</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>URW Gothic L</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Limpar</string>
   </property>
  </widget>
  <widget class="QFrame" name="gpbUsuario">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>381</width>
     <height>301</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>381</width>
     <height>301</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>381</width>
     <height>301</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">.QLabel, .QPlainTextEdit, .QPushButton, .QComboBox, .QRadioButton, .QDateEdit, .QLineEdit
{
	font: 12pt &quot;URW Gothic L&quot;;;
}

QLineEdit
{
	background-color: rgb(0, 0, 0, 100);
}

QComboBox
{	
	background-color: rgb(45, 45, 45);
}

.QPushButton
{	
	background-color: rgb(65, 65, 65, 150);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="lblCamposObrigatorios">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>21</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>URW Gothic L</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Campos obrigatórios*</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtNome">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>171</width>
      <height>32</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>341</width>
      <height>32</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Nome de logon*</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtSenha">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>171</width>
      <height>32</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>341</width>
      <height>32</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Senha*</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lblIndicadorSenha">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>21</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>URW Gothic L</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Senha fraca</string>
    </property>
   </widget>
   <widget class="QLabel" name="ptbImagem">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>121</width>
      <height>121</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>121</width>
      <height>121</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>URW Gothic L</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Imagens/Others/Funcionario.png) 0 0 0 0 stretch stretch;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblImagem">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>21</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>URW Gothic L</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Imagem de Logon</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbPerfil">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>171</width>
      <height>32</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>171</width>
      <height>32</height>
     </size>
    </property>
    <item>
     <property name="text">
      <string>Perfil de usuário*</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Administrador</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Parcial</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="btnCarregar">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>200</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>121</width>
      <height>41</height>
     </size>
    </property>
    <property name="text">
     <string>Carregar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtConfirmeSenha">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>171</width>
      <height>32</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>341</width>
      <height>32</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Confirme a senha *</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txtNome</tabstop>
  <tabstop>txtSenha</tabstop>
  <tabstop>txtConfirmeSenha</tabstop>
  <tabstop>cmbPerfil</tabstop>
  <tabstop>btnCarregar</tabstop>
  <tabstop>btnCancelar</tabstop>
  <tabstop>btnLimpar</tabstop>
  <tabstop>btnConcluir</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
